src/platform/user/profile/utilities/index.js

```diff
@@ -131,7 +131,8 @@ export function mapRawUserDataToState(json) {
 // as a trigger to properly update any components that subscribe to it.
 export const hasSession = () => localStorage.getItem('hasSession');
 
-export const hasSessionSSO = () => localStorage.getItem('hasSessionSSO');
+export const hasSessionSSO = () =>
+  localStorage.getItem('hasSessionSSO') === 'true';
```

Comment:
Is this distinction useful? Why would this get set to some other value rather than get wiped from `localStorage` when the session ends?

Reply:
Either way, I think it'd be useful to clarify how this property behaves.

Reply:
@U-DON I initially thought that having distinct values would be required for login with auto-login/logout on SSO sessions. I'm addressing that and refactoring in a separate PR today, so I'll keep that out of the scope of this PR if that's ok